clonelists/Sony - PlayStation 2.json

```diff
@@ -71,7 +75,8 @@
 	},
 
 	"removes": {
-        "King of Fighters 2000-2001, The (Europe)": {}
+		"King of Fighters 2000-2001, The (Europe)": {
```

Comment:
I'd prefer the braces to be kept together, so it's immediately apparent that it's an empty object :)